### PR TITLE
Introduce a name generator

### PR DESCRIPTION
**Pull request by dariadomagala**
**Description**

Changes proposed in this pull request:

- ...
- ...
- ...

**Related issue(s)**

Resolves https://github.com/kyma-project/busola/issues/874

### CODE REVIEW

File: core-ui/src/shared/utils/randomNamesGenerator/words.js [hunk 1]
```diff
@@ -361,7 +356,6 @@ export const adjectives = [
   'far-off',
   'fast',
   'fat',
```
Comment (akucharska) [inline core-ui/src/shared/utils/randomNamesGenerator/words.js:358]: I would also remove fat

**Review by akucharska — COMMENTED** [review]
Very good job. I have just few more proposition for removal: 
broken, corrupt, critical, damaged, embarrassed, fake, harmful, horrible, lame, loathsome, naive, pitiful, sinful

**Review by mrCherry97 — CHANGES_REQUESTED** [review]
Generating random name should be disabled on `Edit` panel.
image
image
The same is in Jobs when I click generate on Container name:
image
I think in ReplicaSets we have to allow generate names for containers as in Jobs
image
Gateways -> Ports I think the same comment as ReplicaSets
image
RoleBindings and ClusterRoleBindings don't have a name generator
image

**Review by mrCherry97 — CHANGES_REQUESTED** [review]
```
src/components/Predefined/Create/AddonsConfigurations/AddonsConfigurations.js
@kyma-project/core-ui:   Line 3:10:  'MessageStrip' is defined but never used  no-unused-vars
@kyma-project/core-ui:   Line 6:10:  'K8sNameField' is defined but never used  no-unused-vars
@kyma-project/core-ui: src/shared/components/Deployment/Containers.js
@kyma-project/core-ui:   Line 15:14:  'i18n' is assigned a value but never used  no-unused-vars
```

File: core-ui/src/components/Predefined/Create/RoleBindings/RoleBindings.scss [hunk 2]
```diff
@@ -0,0 +1,8 @@
+.binding-name-field {
+  display: flex;
+
+  .fd-input {
+    padding-top: 0;
+    margin-top: 0;
+  }
+}
```
[the remark this answers is not in the document]
Comment (mrCherry97) [reply core-ui/src/components/Predefined/Create/RoleBindings/RoleBindings.scss:8]: You can use `.fd-display-flex` instead custom className with the same functionality.

File: core-ui/src/components/Predefined/Create/AddonsConfigurations/AddonsConfigurations.js [hunk 3]
```diff
@@ -111,24 +118,39 @@ export const AddonsConfigurations = ({
       noValidate
     >
       <div className="fd-form-group">
-        <div className="fd-form-item">
-          <K8sNameInput
+        <FormLabel
+          style={{ display: 'block' }}
```
Comment (mrCherry97) [inline core-ui/src/components/Predefined/Create/AddonsConfigurations/AddonsConfigurations.js:122]: There is a problem with styling in Addons
image

**Review by mrCherry97 — CHANGES_REQUESTED** [review]

**Review by mrCherry97 — APPROVED** [review]

**Review by Wawrzyn321 — APPROVED** [review]

**Pull request merged**